app/src/main/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksFragment.kt

```diff
@@ -159,7 +158,7 @@ class TasksFragment : Fragment() {
     private fun setupListAdapter() {
         val viewModel = viewDataBinding.viewmodel
```

Comment:
Why have this check? Why use val viewModel when it's already an instance variable?